argonaut/src/main/scala/org/http4s/argonaut/ArgonautInstances.scala

```diff
@@ -60,9 +63,37 @@ trait ArgonautInstances {
   }
 }
 
+sealed case class ArgonautInstancesBuilder private[argonaut] (
```

Comment:
If we make this class abstract, we can add more fields in future revisions without breaking binary compatibility.